disk/datadog_checks/disk/disk.py

```diff
@@ -45,10 +45,12 @@ def __init__(self, name, init_config, instances):
         self._custom_tags = instance.get('tags', [])
         self._service_check_rw = is_affirmative(instance.get('service_check_rw', False))
         self._min_disk_size = instance.get('min_disk_size', 0) * 1024 * 1024
+        self._blkid_cache_file = instance.get('blkid_cache_file', '')
 
         self._compile_pattern_filters(instance)
         self._compile_tag_re()
         self._blkid_label_re = re.compile('LABEL=\"(.*?)\"', re.I)
+        self._blkid_device_re = re.compile('>(.*?)</device>', re.I)
```

Comment:
It seems more suitable here to use a xml parser instead of regex. Mainly for parsing reliability. 

You can use that for example https://docs.python.org/2.7/library/xml.etree.elementtree.html